src/main/drivers/bus_spi_ll.c

```diff
@@ -233,27 +233,60 @@ bool spiIsBusBusy(SPI_TypeDef *instance)
 
 bool spiTransfer(SPI_TypeDef *instance, const uint8_t *txData, uint8_t *rxData, int len)
 {
-    uint16_t spiTimeout = 1000;
+    // set 16-bit transfer
+    CLEAR_BIT(instance->CR2, SPI_RXFIFO_THRESHOLD);
```

Comment:
It's possible but pointless. This is only called once from within the code and there `len=7`. Indeed it's pointless calling this function unless len>=2, since there is a transferbyte function.